### DISCUSSION TITLE
Labels Management

### DISCUSSION THREAD

**Discussion by edalcin** [TriliumNext/Trilium#3009] [Q&A]
Hi there!

There is any way to see and manage all the labels?

Thanks in advance

Eduardo

**Comment by zadam** [comment]
Hi, in the demo document there's a "Attribute count statistic". It's meant as a scripting showcase, but could be useful to you:

![image](https://user-images.githubusercontent.com/617641/180646904-9f19d5d5-15cb-4540-bfa8-fc9655f5644f.png)

Not sure what do you mean by the "management" part, what would you like to have as management functionality?

**Reply by edalcin** [reply]
Nice! I don't have the DEMO anymore... How do I get it?

Thanks in advance!

Eduardo

**Reply by edalcin** [reply]
I got this also at the MySQL Console:

`select distinct name from attributes where type='label' order by name;`

From here: https://github.com/zadam/trilium/issues/1346

**Reply by zadam** [reply]
You can import the demo document from this file: https://github.com/zadam/trilium/blob/master/db/demo.zip

Just right click onto a destination, import and select the ZIP. Uncheck "safe import" to make sure scripts work.

**Reply by edalcin** [reply]
Thanks!
